lib/api/src/rest/schema.rs

```diff
@@ -64,26 +64,66 @@ pub enum ShardKeySelector {
     // ToDo: select by pattern
 }
 
+fn id_example() -> segment::types::PointIdType {
+    segment::types::PointIdType::Uuid(uuid::Uuid::new_v4())
+}
+
```

Comment:
Hi @generall,

I'm having trouble figuring out why my example isn't being generated for `id` in `openapi.json`. Could you please help me with this?

Thanks!